### PR TITLE
add reviewer page listing add-ons for a developer

### LINKED ISSUE #20234: Expose add-on ownership information to reviewers
Reviewers should be able to see a list of all add-ons a user owns or owned, with the link to the respective add-on review pages.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #20234 

Looks like this:
Screenshot 2023-02-15 21 44 21
(ugly but functional)

### CODE REVIEW

File: src/olympia/reviewers/templates/reviewers/developer_profile.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  <table>
+    <tbody>
+      <thead>
+        <tr class="listing-header">
+          <th>ID: Name</th>
+          <th>Add-on ID</th>
+          <th>Type</th>
+          <th>Status</th>
+          <th>Role</th>
+          <th>&nbsp;</th>
```
Comment (bobsilverberg) [inline src/olympia/reviewers/templates/reviewers/developer_profile.html:26]: So we really need these `&nbsp;`s?

File: src/olympia/reviewers/templates/reviewers/review.html [hunk 2]
```diff
@@ -24,7 +24,12 @@ <h2 class="addon"{{ addon.name|locale_html }} id="addon-name">
       <span class="version">{{ version.version }} ({{ version.get_channel_display() }})</span>
     {% endif %}
   </h2>
-  <h4 class="author">by {{ users_list(addon.listed_authors) }}</h4>
+  <h4 class="author">by
+    {% for author in addon.listed_authors %}
+    <a href="{{ url('reviewers.developer_profile', author.id) }}">{{ author.name }}</a>
```
Comment (bobsilverberg) [inline src/olympia/reviewers/templates/reviewers/review.html:29]: Does this render the same thing as `users_list`? I.e., do we need some separators or any markup between the authors?

Comment (bobsilverberg) [reply src/olympia/reviewers/templates/reviewers/review.html:29]: Also, do you think we should have a test for this (I don't see one added)?

**Review by bobsilverberg — APPROVED** [review]
Looks good. Just a couple of questions/comments.

r+wc

File: src/olympia/reviewers/templates/reviewers/developer_profile.html (hunk 1 above)
Comment (eviljeff) [reply src/olympia/reviewers/templates/reviewers/developer_profile.html:26]: I thought we'd need `<th>&nbsp;</th>` so the header matched the number of cells in the rows below but I'm not sure if either the &nbsp; or the `<th>` itself are strictly needed

File: src/olympia/reviewers/templates/reviewers/review.html (hunk 2 above)
Comment (eviljeff) [reply src/olympia/reviewers/templates/reviewers/review.html:29]: mmm, I think I'm missing the `, `.  I'll add.

**Pull request merged**